src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -52,8 +52,19 @@ void CaptureViewElement::UpdatePrimitives(Batcher& batcher, TextRenderer& text_r
   batcher.PopTranslation();
 }
 
+bool CaptureViewElement::OnMouseWheel(const Vec2& mouse_pos, int delta,
+                                      const ModifierKeys& modifiers) {
+  return false;
+}
+
 void CaptureViewElement::UpdateLayout() {
+  // Call all methods that may invoke layout changed for children
+  SetWidth(width_);
```

Comment:
See my comments below. I would like to understand the real problem here and have a nicer solution.